docs/clusters/alpine/allocations.md

```diff
@@ -43,23 +43,25 @@ resources requested, job size, and QOS.
 ### Allocations
 When you receive a Research Computing account you are automatically 
 assigned a **Trailhead Auto-Allocation**, which grants you a fixed share 
-of `ucb-general`. The Trailhead is a great allocation for smaller jobs or 
+of `ucb-general`. RMACC users are granted a fixed share of `rmacc-general`. 
+The Trailhead is a great allocation for smaller jobs or 
 testing and benchmarking your code. However, if you consume more than your 
-fixed share of `ucb-general`,  your `LevelFS` will decrease, causing your 
+fixed share of `ucb-general` (or `rmacc-general`),  your `LevelFS` will decrease, causing your 
 priority score to decrease. The end result is that your jobs will sit 
-lower (read: longer) in the queue relative to other jobs. One way to 
+lower in the queue relative to other jobs. One way to 
 combat this is to apply for an allocation.
 
 ## Alpine Allocation Tiers
 
-In addition to the Trailhead auto-allocation (`ucb-general`) that all users are awarded automatically, CURC offers two 
+In addition to the Trailhead auto-allocation (`ucb-general` or `rmacc-general`) that all users are awarded automatically, CURC offers two 
 additional tiers to accommodate larger computing needs on Alpine. The **Ascent Allocation** tier provides users 
 with 350,000 SUs over a 12 month period. The **Peak Allocation** tier is 
 aimed at projects that will consume between 350,000 and 6,000,000 SUs in a 
 12 month period. Users may apply for these tiers as described below.
+RMACC users are not eligible for Ascent or Peak allocations, but may apply for an **RMACC Allocation**, which provides users with 100,000 SUs over a 12 month period.
```

Comment:
Note we need to change the RMACC form so that it doesn't say Ascent anymore.

Reply:
At this time, no Peak allocations for RMACC users. Keep "Ascent" wording.
"CURC does not currently support Peak allocations for RMACC users. However, please contact us if you would like to discuss..."